### PR TITLE
Social share

### PR DESCRIPTION
**Pull request by uditnaagar777**
Social sharing options added on all products pages

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js [hunk 1]
```diff
@@ -27,6 +27,15 @@ $(document).ready(function() {
     let productDetail;
     try {
      productDetail = JSON.parse(this.responseText);
+       let swfMailTo = document.getElementById('swfMailTo');
+ swfMailTo.href = "mailto:?"
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js]: @uditnaagar777 plz remove js related to share component from pdp js and include the js in clientlibs of share component

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/less/productdetails.less [hunk 2]
```diff
@@ -175,4 +175,37 @@
...
 		}
 	}
+}
+
+
+.fa-twitter {
+ background: white;
+ color: #55ACEE;
+ }
+ .fa-facebook-official{
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/less/productdetails.less]: @uditnaagar777 plz remove css related to share component from pdp less file and include that in clientlibs of share component

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
 	<div class='product-details-cmp__img aem-Grid aem-Grid--9 aem-Grid--default--12'>
 		<img class='product-details-cmp--product-img' src="${properties.fileReference}" alt="image" />
-	</div>
-	<div class='product-details-cmp__product-details aem-Grid aem-Grid--3 aem-Grid--default--12'> <b class='product-details-cmp__prduct-name'>${properties.productName} </b>
-		<p class='product-sku'>${properties.productSku}</p>
+	<div>
+            <div class="productName">
+                <b class='product-details-cmp__prduct-name'>${properties.productName} </b>
+            </div>
+               <div data-sly-resource="${'SocialShare' @ resourceType='hclecomm/components/content/SocialShare', decoration=true}"></div>
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html]: @uditnaagar777 plz remove share component  from pdp component as we can author it beside title component as separate component

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js (hunk 1 above)
Comment (uditnaagar777) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js]: This is done. Thanks for the review

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/less/productdetails.less (hunk 2 above)
Comment (uditnaagar777) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/less/productdetails.less]: This is done. Thanks for the review

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html (hunk 3 above)
Comment (uditnaagar777) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html]: As of now , we dont have title component. We have created a seperate component for social share. Once we get the title component & there is any UI issue, we will update CSS accordingly

**Pull request closed without merging**